ckanext/saml2auth/views/saml2auth.py

```diff
@@ -76,8 +76,7 @@ def acs():
         # If there is not configuration set up for config for organisation_mapping or read_only_saml_groups, it will return True to carry on login workflow
         if not h.saml_group_mapping_exist(groups):
             log.warning('User {0} {1} groups {2} does not exists'.format(firstname, lastname, groups))
-            toolkit.h.flash_error(toolkit._('Not authorized'))
-            return toolkit.h.redirect_to('user.login')
+            return toolkit.h.redirect_to('saml2auth.unauthorised', firstName=firstname, lastName=lastname, email=email)
```

Comment:
@salsa-nathan I noticed in the above method `disable_default_login_register` that it used the template `error_document_template.html` so decided to use the same template to display the unauthorised message. Let me know if this fits the requirements or to create a new template?